### PR TITLE
macOS build script fixes

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: scripts/macos_build_wheel.py [hunk 1]
```diff
@@ -50,8 +50,6 @@ def get_python_info():
 def build_wheel(cleanup=False):
     py_exe, py_ver, py_inc_dir, py_lib = get_python_info()
     build_type = 'Release'
-    source_path = "%s/%s-source" % (STANDALONE_DIR, PROJECT_NAME)
-    build_path = "%s/%s-osx_%s" % (ROOT_DIR, PROJECT_NAME, py_ver)
```
Comment (jcfr) [inline scripts/macos_build_wheel.py]: Similarly to what is done on manylinux (see [here](https://github.com/KitwareMedical/VTKPythonPackage/blob/5d217a5fb9b49146c3229488dd63410e2adc4f7c/scripts/internal/manylinux-build-wheels.sh#L17-L22)), the build process may be speed up by re-using the build tree.

Comment (thewtex) [reply scripts/macos_build_wheel.py]: :+1: Issue #31 created

Comment (thewtex) [reply scripts/macos_build_wheel.py]: I also noted these todos:

  https://github.com/KitwareMedical/VTKPythonPackage/issues/32

**Comment by jcfr** [comment]
@maekclena Could you check if this PR as well as #32 should be closed ?

**Comment by maekclena** [comment]
These changes just need to be rebased, if you want I can open a PR for it. For issue #32, PR #34 implements all items on the list so it can be closed.

**Comment by thewtex** [comment]
@maekclena thanks so much for your help on this. This will help us add `itk-vtkglue` packages for macOS, too. A rebase and new PR would be appreciated. :1st_place_medal: 

CC: @fbudin69500

**Comment by jcfr** [comment]
Closing. This was superseded by #35

**Pull request closed without merging**